src/Sylius/Component/Inventory/Factory/InventoryUnitFactory.php

```diff
@@ -17,8 +17,6 @@
 use Sylius\Component\Resource\Factory\Factory;
 
 /**
- * Default inventory operator.
- *
  * @author Paweł Jędrzejewski <pawel@sylius.org>
  */
 class InventoryUnitFactory extends Factory implements InventoryUnitFactoryInterface
```

Comment:
I thought we're using it somewhere else, not in *CoreBundle*, but after quick search we don't, so sure, can be removed ;)